cache/volume_cache_test.go

```diff
@@ -0,0 +1,353 @@
+package cache_test
+
+import (
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpack/lifecycle"
+	"github.com/buildpack/lifecycle/cache"
+
+	h "github.com/buildpack/lifecycle/testhelpers"
```

Comment:
consolidate the last two import groups

Reply:
@sclevine How did you hack into Emily's account...? ;P

Reply:
Imports fixed.